src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ docker:
       address: ${IEXEC_WORKER_DOCKER_REGISTRY_ADDRESS_1:}
       username: ${IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_1:}
       password: ${IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_1:}
+  image:
+    pull-timeout:
+      min: ${IEXEC_WORKER_DOCKER_IMAGE_MIN_PULL_TIMEOUT:5}   # image min pull timeout in minutes
```

Comment:
Since we generally use durations in seconds ([exp](https://github.com/iExecBlockchainComputing/iexec-core/blob/develop/src/main/resources/application.yml#L22)), I suggest we do the same thing here to be consistent, and add a comment that mentions it in minutes.

Reply:
@jeremyjams suggested [here](https://github.com/iExecBlockchainComputing/iexec-worker/pull/442#discussion_r834461584) that we should strive to have a single unit. I chose minutes as that makes calculations easier. So either we stick with minutes everywhere for this context, either we get back to seconds - which are less easier to understand I think.

Reply:
If we keep minutes for a better admin UX, we need to specify it explicitly, something like:
`IEXEC_WORKER_DOCKER_IMAGE_MIN_PULL_TIMEOUT_MINUTES`

Reply:
We now use a special format for `Duration`, no need to precise its unit. See https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/75c5434b0ec23721e8f55b178adac6eb4277a833.

Reply:
Thank @mcornaton, let's make sure the format is well documented/understood for the DevOps team. Other that well done.